src/wymeditor/core.js

```diff
@@ -775,17 +775,30 @@ jQuery.fn.wymeditor = function (options) {
         preInit: null,
         preBind: null,
         postInit: null
+    }
+
+    mergedOptions = jQuery.extend(defaultOptions, options);
 
-    }, options);
+    // If an iframeHtml option wasn't included, default to using the iframeHtml
+    // defined by the skin.
+    if (typeof options.iframeHtml !== "undefined") {
```

Comment:
I would write
```js
if (!options.iframeHtml) {
}
```
It is a bit easier on the eyes, for me. It also resolves to `true` when `options.iframeHtml === ''` (an empty string).
![screenshot_2015-04-24_13-02-06](https://cloud.githubusercontent.com/assets/635591/7316729/27f9eb0e-ea82-11e4-80c2-4adc3ecfeb25.png)